### PR TITLE
Add tvOS platform to RTDB

### PR DESCRIPTION
**Pull request by judyliu14**
TODO:
- [x] Fix navigation formatting issue after creating a new post

### CODE REVIEW

**Comment by ncooke3** [comment]
I tested the latest commit on Xcode 13's latest beta and the layout issue seems to have been resolved. I looked at the tvOS beta release notes and [this note may have addressed the issue](https://developer.apple.com/documentation/tvos-release-notes/tvos-15-beta-release-notes#:~:text=List%20now%20correctly%20respects%20safe%20area%20insets.%20(78833004)).

**Comment by judyliu14** [comment]
> I tested the latest commit on Xcode 13's latest beta and the layout issue seems to have been resolved. I looked at the tvOS beta release notes and [this note may have addressed the issue](https://developer.apple.com/documentation/tvos-release-notes/tvos-15-beta-release-notes#:~:text=List%20now%20correctly%20respects%20safe%20area%20insets.).

That sounds good, this means it should work properly with the latest xcode version.

File: database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/NewPostsView.swift [hunk 1]
```diff
@@ -24,6 +24,11 @@ struct NewPostsView: View {
 
   var body: some View {
     HStack {
+      Button(action: {
+        isPresented = false
+      }) {
+        Image(systemName: "chevron.left")
+      }
```
Comment (ncooke3) [inline database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/NewPostsView.swift:31]: Double checking, was this new Button view intended to appear on all platforms or just tvOS?

**Review by ncooke3 — APPROVED** [review]
LGTM, great work! Left one question above 👆

File: database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/NewPostsView.swift (hunk 1 above)
Comment (judyliu14) [reply database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/NewPostsView.swift:31]: the new button appears on all platforms

**Pull request merged**